vizro-ai/docs/pages/user_guides/run_vizro_ai.md

```diff
@@ -80,3 +80,19 @@ Vizro-AI's `_get_chart_code` method returns the Python code string that can be u
 
 The returned `code_string` can be used to dynamically render charts within your application. You may have the option to encapsulate the chart within a `fig` object or convert the figure into a JSON string for further integration.
 In case you would like to use the insights or code explanation, you can use `vizro_ai._run_plot_tasks(df, ..., explain=True)`, which returns a dictionary containing the code explanation and chart insights alongside the code.
+
+### 4. Increase the maximum number of attempts to debug
+Sometimes there is error in code generated first round since the code is generated by Large Language Models (LLMs),
```

Comment:
```suggestion
Since the code is generated by Large Language Models (LLMs), it can contain bugs and thus fail on execution. In this case, one can use the `max_debug_retry` parameter to specify the maximum desired number of debugging attempts.
```